tests/output.rs

```diff
@@ -23,24 +23,29 @@ fn get_fixture_dir_path_from_name(fixture_dir_name: &str) -> PathBuf {
 
 fn parse_command_and_output(command_and_output: &str) -> CommandAndOutput {
     let mut lines = command_and_output.split('\n').collect::<Vec<_>>();
-    if !lines.is_empty() {
-        if regex!(r#"^\s*$"#).is_match(lines[0]) {
-            lines.remove(0);
-        }
+    if lines.is_empty() {
+        panic!("Expected at least a command line");
+    }
+    if lines[0].trim().is_empty() {
+        lines.remove(0);
     }
     let command_line = lines.remove(0);
     let indent = regex!(r#"^\s*"#).find(command_line).unwrap().as_str();
     let command_line_args = parse_command_line(strip_indent(command_line, indent));
     if !lines.is_empty() {
-        if regex!(r#"^\s*$"#).is_match(lines[lines.len() - 1]) {
+        if lines[lines.len() - 1].trim().is_empty() {
             lines.pop();
         }
     }
     let output: String = lines
         .into_iter()
         .map(|line| {
-            assert!(line.starts_with(indent));
-            format!("{}\n", strip_indent(line, indent))
+            if line.is_empty() {
+                "\n".to_owned()
```

Comment:
This is just to not have to add `indent` spaces at the beginning of blank output lines (since my editor wasn't inclined to do that)